Codeinwp/neve
# enh: redirect customizer preview to blog page when entering Blog/Archive Section

### Summary
Sets the customizer preview to the blog page when entering the blog/archive section.


### Will affect visual aspect of the product

NO

### Test instructions


- Should change preview URL to blog in all cases, even when you change the homepage settings inside the customizer


Closes #1988.
